packages/manifest/src/pixiManifest.ts

```diff
@@ -100,15 +133,19 @@ function collectAssets(
             {
                 bundleAssets.push({
                     alias: getShortNames(stripTags(path.relative(entryPath, `${asset.path}-${pageIndex}`)), options),
-                    src: pages.map((finalAsset) => path.relative(outputPath, finalAsset.path))
+                    src: pages
```

Comment:
it doesn't seem like we are adding `data` anymore?

we used to add all the tags to the manifest